### DISCUSSION TITLE
Consiglio per multipresa smart e caricabatterie cellulari

### DISCUSSION THREAD

**Discussion by roby70cr** [indomus/forum#2278] [HUB: Home Assistant]
Buongiorno,
vorrei prendere una multipresa smart per usarla come stazione di ricarica per il cellulare, smartwatch, powerbank, e altri dispositivi all'occorrenza.
Mi piacerebbe che le singole prese si potessero attivare/disattivare a seconda dello stato di carica delle batterie dei singoli dispositivi, e integrare il tutto in Homeassistant.
E' qualcosa di fattibile? Sono alle prime armi con il sistema, a fatica mi sto facendo strada nei meandri delle varie configurazioni con alternanti risultati, avrei bisogno di un consiglio per cominciare, intanto sull'hardware. Meglio un prodotto già confezionato (tipo Meross) ma non sono un fan dell'app smartlife/tuya, oppure meglio prendere una board con relè da collegare a una multipresa standard che magari sia ewelink compatibile?
Seguirà poi la parte di configurazione in HA, così da avere i caricabatterie spenti normalmente, e che si accendono quando il telefono è collegato in carica e la batteria sia sotto il 15%, e che si spengono al raggiungimento del 90, ognuno chiaramente indipendente.
Grazie a chi vorrà instradarmi.

**Comment by sky7176** [comment]
Ciao,
io userei una ciabatta Smart della lidl, è zigbee ed ha tre prese comandabili singolarmente.
È intrgrabile in HA.
Io la uso senza difficoltà, devi solo aspettare che la rimettano in vendita, altrimenti se cerchi su Amazon trovi diversi articolo, magari fai la ricerca specificando home assistant.

Giuseppe

**Comment by roby70cr** [comment]
grazie della risposta, ho visto qualche ciabatta ma la maggior parte è smartlife/tuya.
usi anche automazioni?

**Reply by sky7176** [reply]
Io le faccio con node-red
